Justfile

```diff
@@ -0,0 +1,4 @@
+# Generate the Go protobuf using the Docker build (writes pb file to repo root).
+protogen:
```

Comment:
It looks like a simple shell script would be a better solution here: no need to install the `just` tool.